auth0_flutter_platform_interface/lib/src/method_channel_auth0_flutter_auth.dart

```diff
@@ -20,6 +21,10 @@ const MethodChannel _channel = MethodChannel('auth0.com/auth0_flutter/auth');
 const String authLoginMethod = 'auth#login';
 const String authLoginWithOtpMethod = 'auth#loginOtp';
 const String authMultifactorChallengeMethod = 'auth#multifactorChallenge';
+const String authPasswordlessWithEmailMethod = 'auth#passwordlessWithEmail';
+const String authPasswordlessWithPhoneNumberMethod = 'auth#passwordlessWithPhoneNumber';
+const String authLoginWithEmail = 'auth#loginWithEmail';
+const String authLoginWithPhoneNumber = 'auth#loginWithPhoneNumber';
```

Comment:
These should be:
- `authLoginWithEmailMethod`
- `authLoginWithPhoneNumberMethod`

Reply:
Unless the login methods themselves are renamed as suggested, in which case these should be:
- `authLoginWithEmailCodeMethod`
- `authLoginWithSMSCodeMethod`